test/publish/test_openaccess.py

```diff
@@ -110,6 +110,7 @@ def test_write_contributions(test_session, snapshot, dataset, caplog):
     assert row.sunet == "janes"
     assert row.role == "faculty"
     assert bool(row.academic_council) is True
+    assert bool(row.faculty_authored) is True
```

Comment:
This is a bit of a nit pick, but could we have `faculty` instead of `faculty_authored`. We don't have `academic_council_authored` or `role_authored`, etc...

Reply:
Actually we do have an `academic_council_authored` ticket. We don't need a `role_authored` column because we don't use that in our charts. But my concern with `faculty` vs `faculty_authored` is its easy to get confused with what the data actually means. `faculty_authored` is clear to me and the name implies that the data is about the publication. `academic_council` is another column that is connected to a person (is that person a member of the academic council). If we name it `faculty` it could lead us to falsely assume that a `TRUE` value in that column means the author is a faculty member. We can talk through this at standup if need be.

Reply:
Ok, consistency is important, so perhaps this PR could address those other naming issues?